prime-router/metadata/tables/LIVD-SARS-CoV-2-old-1.csv

```diff
@@ -0,0 +1,4713 @@
+Manufacturer,Model,Vendor Analyte Name,Vendor Specimen Description,Vendor Result Description,Test Performed LOINC Code,Test Performed LOINC Long Name,Test Ordered LOINC Code,Test Ordered LOINC Long Name,Vendor Comment,Vendor Analyte Code,Vendor Reference ID,Testkit Name ID,Testkit Name ID Type,Equipment UID,Equipment UID Type,Component,Property,Time,System,Scale,Method,Publication Version ID,LOINC Version ID
```

Comment:
From the name, it sounds like this file should be deleted.  All other tables have a date.  I do not think this should be checked in